examples/nlp/text_classification/text_classification_with_bert.py

```diff
@@ -0,0 +1,207 @@
+    parser.add_argument("--eval_file_prefix", default='dev', type=str, help="eval file prefix")
+    parser.add_argument("--no_shuffle", action='store_false', dest="shuffle", help="Shuffle is enabled by default.")
+    parser.add_argument("--num_train_samples", default=-1, type=int, help="Number of samples to use for training")
+    parser.add_argument("--num_val_samples", default=-1, type=int, help="Number of samples to use for evaluation")
+    parser.add_argument("--num_workers", default=2, type=int, help="The number of workers for the data loaders.")
+    parser.add_argument(
+        "--pin_memory", action='store_true', help="Whether to enable the pin_memory feature of the data loaders."
+    )
+    parser.add_argument(
+        "--use_cache", action='store_true', help="When specified loads and stores cache preprocessed data."
+    )
+
+    # BERT Arguments
+    parser.add_argument(
+        '--pretrained_model_name',
+        default='roberta-base',
+        type=str,
+        help='Name of the pre-trained model',
+        # choices=nemo_nlp.modules.common.get_pretrained_lm_models_list(),
+    )
+    parser.add_argument("--bert_checkpoint", default=None, type=str, help="Path to pre-trained BERT checkpoint")
+    parser.add_argument("--bert_config", default=None, type=str, help="Path to bert config file in json format")
+    parser.add_argument(
+        "--tokenizer",
+        default="nemobert",
+        type=str,
+        choices=["nemobert", "sentencepiece"],
+        help="tokenizer to use, only relevant when using custom pretrained checkpoint.",
+    )
+    parser.add_argument("--vocab_file", default=None, type=str, help="Path to the vocab file.")
+    parser.add_argument(
+        "--tokenizer_model",
+        default=None,
+        type=str,
+        help="Path to pretrained tokenizer model, only used if --tokenizer is sentencepiece",
+    )
+    parser.add_argument(
+        "--do_lower_case",
+        action='store_true',
+        help="Whether to lower case the input text. True for uncased models, False for cased models. "
+        + "For tokenizer only applicable when tokenizer is build with vocab file.",
+    )
+    parser.add_argument("--batch_size", default=32, type=int, help="Training and evaluation batch size")
+    parser.add_argument(
+        "--max_seq_length",
+        default=36,
+        type=int,
+        help="The maximum total input sequence length after tokenization.Sequences longer than this will be \
+                        truncated, sequences shorter will be padded.",
+    )
+
+    # Model Arguments
+    parser.add_argument("--num_output_layers", default=2, type=int, help="Number of layers in the Classifier")
+    parser.add_argument("--fc_dropout", default=0.1, type=float, help="Dropout rate")
+    parser.add_argument("--class_balancing", default="None", type=str, choices=["None", "weighted_loss"])
+
+    # Training Arguments
+    parser.add_argument("--num_gpus", default=1, type=int, help="Number of GPUs")
```

Comment:
please use "gpus" to keep it consistent with PTL

Reply:
Done.